export_job/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM quay.io/cdis/python:3.7-stretch
+FROM quay.io/cdis/python:3.8-buster
 
-RUN pip install git+https://github.com/uc-cdis/gen3sdk-python.git@feat/drs_download_federated_manifest
+RUN pip install git+https://github.com/uc-cdis/gen3sdk-python.git@ca58b94ebcc1377122da10b716d7a01722cfc69b
```

Comment:
Are these features on SDK master now?
I think this job should remain on a feature branch so long as it's using SDK code that's also on a branch. 
That's just my 2 cents though, these changes seem fine to me